charts/zookeeper-operator/values.yaml

```diff
@@ -47,3 +47,18 @@ affinity: {}
 tolerations: []
 # Pod annotations
 annotations: {}
+
+## AdditionalSidecar configuration details.
+additionalSidecars: {}
+# - name: nginx
+#   image: nginx:latest
+
+## AdditionalSidecarEnvironment configuration details.
+additionalEnv: {}
```

Comment:
```suggestion
additionalSidecarEnv: {}
```

Reply:
These are env's for the zookeeper container, not the sidecars. The comment and docs are misleading

Reply:
Updated additionalEnv to additionalSidecarEnv.

Reply:
@amuraru:
Can you see the @azun comments regarding the renaming the additionalEnv to additionalSidecarEnv.
Wdyt?